src/PolarSSL++/RsaPrivateKey.cpp

```diff
@@ -100,7 +100,7 @@ AString cRsaPrivateKey::GetPubKeyDER(void)
 	{
 		return AString();
 	}
-	return AString((const char *)(buf + sizeof(buf) - res), (size_t)res);
+	return AString(static_cast<const char *>(static_cast<const void*>(buf + sizeof(buf) - res)), static_cast<size_t>(res));
```

Comment:
Both should work and I'll change it to `reinterpret_cast`; however in my opinion using the weakest cast that does the job (however ugly it was) would be the conservative choice.